### PR TITLE
Fixes missing access for the raider rigs

### PR DESCRIPTION
**Pull request by Arrow768**

### CODE REVIEW

File: maps/aurora/aurora-1_centcomm.dmm [hunk 1]
```diff
@@ -17967,7 +17967,9 @@
 /obj/item/weapon/tank/oxygen,
 /obj/item/clothing/shoes/magboots,
 /obj/item/clothing/shoes/magboots,
-/obj/item/weapon/rig/hazard/equipped,
+/obj/item/weapon/rig/hazard/equipped{
+	req_access = newlist()
```
Comment (Sindorman) [inline maps/aurora/aurora-1_centcomm.dmm]: I think it should be null

File: maps/aurora/aurora-1_centcomm.dmm [hunk 2]
```diff
@@ -18035,7 +18037,9 @@
 /obj/item/weapon/tank/oxygen,
 /obj/item/clothing/shoes/magboots,
 /obj/item/clothing/shoes/magboots,
-/obj/item/weapon/rig/eva/equipped,
+/obj/item/weapon/rig/eva/equipped{
+	req_access = newlist()
```
Comment (Sindorman) [inline maps/aurora/aurora-1_centcomm.dmm]: Same thing as above.

File: maps/aurora/aurora-1_centcomm.dmm (hunk 1 above)
Comment (Arrow768) [reply maps/aurora/aurora-1_centcomm.dmm]: Those are the defaults of the industrial rig (which has no access restrictions)
image

Comment (Sindorman) [reply maps/aurora/aurora-1_centcomm.dmm]: Huh, I thought they follow same principal as airlocks and machines which is null.

**Pull request merged**